R/demo-education.R

```diff
@@ -1,20 +1,32 @@
 #' Access to Education Data
 #'
-#' This function returns a data frame from Education in State Cancer Profiles
+#' This function returns a data frame about education demographics from State Cancer Profiles.
 #'
 #' @param area A state/territory abbreviation or USA.
-#' @param areatype Either "county", "hsa" (Health service area), or "state"
-#' @param education Either "less than 9th grade", "at least high school", "at least bachelors degree"
-#' @param sex Either "both sexes", "male", "female"
-#' @param race One of the following values: "All Races (includes Hispanic)", "white (includes hispanic)",
-#'              "white non-hispanic","black","amer. indian/alaskan native (includes hispanic)",
-#'              "asian or pacific islander (includes hispanic)","hispanic (any race)
+#' @param areatype Either `"county"`, `"hsa"` (Health Service Area), or
+#' `"state"`.
+#' @param education The only permissible values are 
```

Comment:
Be consistent with wording in the param descriptions too. I like `One of the following values:`

Reply:
Got it, I will use "One of the following values"

Reply:
are there supposed to periods at the end of params as well?

Reply:
> are there supposed to periods at the end of params as well?

yes